discovery-infra/test_infra/assisted_service_api.py

```diff
@@ -174,13 +174,13 @@ def _download(cls, response: HTTPResponse, file_path: str, verify_file_size=Fals
                 )
 
     def generate_image(
-            self,
-            cluster_id: str,
-            ssh_key: str,
-            image_type: str = consts.ImageType.FULL_ISO,
-            static_network_config: Optional[list] = None,
+        self,
+        cluster_id: str,
+        ssh_key: str,
+        image_type: str = consts.ImageType.FULL_ISO,
+        static_network_config: Optional[list] = None,
     ) -> models.cluster.Cluster:
-        cluster = self.cluster_get(cluster_id=cluster_id)
+        self.cluster_get(cluster_id=cluster_id)
```

Comment:
then why do we do that?
did we wanted to return it at the end?

Reply:
We are not using it.. I think we can just delete it